routers/web/repo/issue.go

```diff
@@ -784,8 +784,10 @@ func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles
 			}
 
 		}
-		if !strings.HasPrefix(template.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
-			template.Ref = git.BranchPrefix + template.Ref
+		if template.Ref != "" {
+			if !strings.HasPrefix(template.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
+				template.Ref = git.BranchPrefix + template.Ref
+			}
```

Comment:
```suggestion
		if template.Ref != "" && !strings.HasPrefix(template.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users 
			template.Ref = git.BranchPrefix + template.Ref
```
Does it make sense to merge these conditions, or is it less clear?